railway build failed

@claude railway build failed. here's the build log: 
2026-04-24T04:08:15.366019123Z [inf]  
2026-04-24T04:08:15.764412961Z [inf]  [35m[Region: us-west1][0m
2026-04-24T04:08:15.781476454Z [inf]  
2026-04-24T04:08:15.781503994Z [inf]  [38;2;125;86;243m╭─────────────────╮[0m
2026-04-24T04:08:15.781508004Z [inf]  [38;2;125;86;243m│[0m Railpack 0.23.0 [38;2;125;86;243m│[0m
2026-04-24T04:08:15.781511307Z [inf]  [38;2;125;86;243m╰─────────────────╯[0m
2026-04-24T04:08:15.781515860Z [inf]  
2026-04-24T04:08:15.781519254Z [inf]  [97m↳ Detected Shell[0m
2026-04-24T04:08:15.781526166Z [inf]  [97m↳ Using shell script: start.sh with interpreter: bash[0m
2026-04-24T04:08:15.781531489Z [inf]  [97m↳ Found web command in Procfile[0m
2026-04-24T04:08:15.781534722Z [inf]  
2026-04-24T04:08:15.781538019Z [inf]  [1mSteps[0m
2026-04-24T04:08:15.781541338Z [inf]  [38;5;238m──────────[0m
2026-04-24T04:08:15.781544023Z [inf]  [95m▸ build[0m
2026-04-24T04:08:15.781547339Z [inf]  [38;5;245m$[0m [1msh build.sh[0m
2026-04-24T04:08:15.781550446Z [inf]  
2026-04-24T04:08:15.781553117Z [inf]  [1mDeploy[0m
2026-04-24T04:08:15.781555942Z [inf]  [38;5;238m──────────[0m
2026-04-24T04:08:15.781559040Z [inf]  [38;5;245m$[0m [1mbash start.sh[0m
2026-04-24T04:08:15.781561806Z [inf]  
2026-04-24T04:08:15.781564624Z [inf]  
2026-04-24T04:08:15.782229298Z [inf]  Successfully prepared Railpack plan for build
2026-04-24T04:08:15.782261906Z [inf]  
2026-04-24T04:08:15.782271417Z [inf]  
2026-04-24T04:08:15.782921158Z [inf]  context: f37b-3AlB
2026-04-24T04:08:15.935144199Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-24T04:08:15.935180679Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-24T04:08:15.935194138Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-24T04:08:15.945956143Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-24T04:08:15.947015480Z [inf]  resolve image config for docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0
2026-04-24T04:08:16.195543671Z [inf]  resolve image config for docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0
2026-04-24T04:08:16.195608452Z [inf]  docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0@sha256:ba4c430961d9ee3215c64807727a4b11e2198daac31250e9db9eaf9cee4624d6
2026-04-24T04:08:16.195623301Z [inf]  docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0@sha256:ba4c430961d9ee3215c64807727a4b11e2198daac31250e9db9eaf9cee4624d6
2026-04-24T04:08:16.200071681Z [inf]  docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0@sha256:ba4c430961d9ee3215c64807727a4b11e2198daac31250e9db9eaf9cee4624d6
2026-04-24T04:08:16.288237685Z [inf]  load build definition from railpack-plan.json
2026-04-24T04:08:16.288285171Z [inf]  load build definition from railpack-plan.json
2026-04-24T04:08:16.288311687Z [inf]  load build definition from railpack-plan.json
2026-04-24T04:08:16.288323159Z [inf]  load build definition from railpack-plan.json
2026-04-24T04:08:16.310351988Z [inf]  load build definition from railpack-plan.json
2026-04-24T04:08:16.462331159Z [inf]  install mise packages: 
2026-04-24T04:08:16.462369632Z [inf]  create mise config
2026-04-24T04:08:16.462377438Z [inf]  mkdir /etc/mise
2026-04-24T04:08:16.462384921Z [inf]  copy mise.toml
2026-04-24T04:08:16.462392966Z [inf]  loading .
2026-04-24T04:08:16.462401308Z [inf]  docker-image://ghcr.io/railwayapp/railpack-builder:mise-2026.3.17
2026-04-24T04:08:16.462406456Z [inf]  docker-image://ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17
2026-04-24T04:08:16.462493159Z [inf]  [railpack] merge ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17, $packages:mise, $build
2026-04-24T04:08:16.462501283Z [inf]  copy /app
2026-04-24T04:08:16.462507030Z [inf]  sh build.sh
2026-04-24T04:08:16.462512576Z [inf]  [railpack] secrets hash
2026-04-24T04:08:16.462526318Z [inf]  copy .
2026-04-24T04:08:16.462534923Z [inf]  copy / /app
2026-04-24T04:08:16.462541618Z [inf]  copy /root/.local/state/mise
2026-04-24T04:08:16.462546869Z [inf]  copy /etc/mise/config.toml
2026-04-24T04:08:16.462552577Z [inf]  copy /usr/local/bin/mise
2026-04-24T04:08:16.462559735Z [inf]  copy /mise/installs
2026-04-24T04:08:16.462565385Z [inf]  copy /mise/shims
2026-04-24T04:08:16.462609009Z [inf]  docker-image://ghcr.io/railwayapp/railpack-builder:mise-2026.3.17
2026-04-24T04:08:16.462618766Z [inf]  docker-image://ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17
2026-04-24T04:08:16.462752994Z [inf]  [railpack] secrets hash
2026-04-24T04:08:16.462789625Z [inf]  loading .
2026-04-24T04:08:16.462812840Z [inf]  [railpack] secrets hash
2026-04-24T04:08:16.660007457Z [inf]  docker-image://ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17
2026-04-24T04:08:16.660316524Z [inf]  docker-image://ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17
2026-04-24T04:08:16.707132522Z [inf]  docker-image://ghcr.io/railwayapp/railpack-builder:mise-2026.3.17
2026-04-24T04:08:16.707481145Z [inf]  [railpack] secrets hash
2026-04-24T04:08:16.707507084Z [inf]  docker-image://ghcr.io/railwayapp/railpack-builder:mise-2026.3.17
2026-04-24T04:08:16.707521385Z [inf]  docker-image://ghcr.io/railwayapp/railpack-builder:mise-2026.3.17
2026-04-24T04:08:16.707527287Z [inf]  loading .
2026-04-24T04:08:16.737549122Z [inf]  loading .
2026-04-24T04:08:16.738917200Z [inf]  copy mise.toml
2026-04-24T04:08:16.768987621Z [inf]  copy mise.toml
2026-04-24T04:08:16.769672541Z [inf]  mkdir /etc/mise
2026-04-24T04:08:16.779811133Z [inf]  mkdir /etc/mise
2026-04-24T04:08:16.780370649Z [inf]  create mise config
2026-04-24T04:08:16.789927196Z [inf]  create mise config
2026-04-24T04:08:16.791142072Z [inf]  install mise packages: 
2026-04-24T04:08:17.841168606Z [inf]  mise python@3.11.15  [1/3] install

2026-04-24T04:08:17.84198641Z [inf]  mise node@20.20.2    [1/3] install

2026-04-24T04:08:17.843022355Z [inf]  mise node@20.20.2    [1/3] download node-v20.20.2-linux-x64.tar.gz

2026-04-24T04:08:17.959769284Z [inf]  mise python@3.11.15  [1/3] download cpython-3.11.15+20260414-x86_64-unknown-linux-gnu-install_only_stripped.tar.gz

2026-04-24T04:08:18.164880647Z [inf]  mise node@20.20.2    [2/3] generate checksum node-v20.20.2-linux-x64.tar.gz

2026-04-24T04:08:18.17904576Z [inf]  mise node@20.20.2    [3/3] extract node-v20.20.2-linux-x64.tar.gz

2026-04-24T04:08:18.47808178Z [inf]  mise python@3.11.15  [1/3] generate checksum cpython-3.11.15+20260414-x86_64-unknown-linux-gnu-install_only_stripped.tar.gz

2026-04-24T04:08:18.488222028Z [inf]  mise python@3.11.15  [1/3] extract cpython-3.11.15+20260414-x86_64-unknown-linux-gnu-install_only_stripped.tar.gz

2026-04-24T04:08:18.870310222Z [inf]  mise node@20.20.2    [3/3] node -v

2026-04-24T04:08:18.873674714Z [inf]  mise node@20.20.2    [3/3] v20.20.2

2026-04-24T04:08:18.874349749Z [inf]  mise node@20.20.2    [3/3] npm -v

2026-04-24T04:08:18.960295232Z [inf]  mise python@3.11.15  [1/3] python --version

2026-04-24T04:08:18.96105069Z [inf]  mise node@20.20.2    [3/3] 10.8.2

2026-04-24T04:08:18.964024487Z [inf]  mise python@3.11.15  [1/3] Python 3.11.15

2026-04-24T04:08:18.970604323Z [inf]  mise python@3.11.15 ✓ installed

2026-04-24T04:08:18.971561618Z [inf]  mise node@20.20.2  ✓ installed

2026-04-24T04:08:19.104143556Z [inf]  install mise packages: 
2026-04-24T04:08:19.104484480Z [inf]  copy / /app
2026-04-24T04:08:19.186012459Z [inf]  copy / /app
2026-04-24T04:08:19.187009345Z [inf]  copy .
2026-04-24T04:08:19.209752013Z [inf]  copy .
2026-04-24T04:08:19.210542039Z [inf]  sh build.sh
2026-04-24T04:08:19.297714411Z [inf]  Building frontend...

2026-04-24T04:08:20.564313449Z [inf]  copy /mise/shims
2026-04-24T04:08:20.570166463Z [inf]  copy /mise/shims
2026-04-24T04:08:20.570920677Z [inf]  copy /mise/installs
2026-04-24T04:08:20.726738604Z [inf]  npm warn deprecated whatwg-encoding@3.1.1: Use @exodus/bytes instead for a more spec-conformant and faster implementation

2026-04-24T04:08:20.980392206Z [inf]  npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported

2026-04-24T04:08:21.475273321Z [inf]  npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.

2026-04-24T04:08:21.492441610Z [inf]  copy /mise/installs
2026-04-24T04:08:21.493313435Z [inf]  copy /usr/local/bin/mise
2026-04-24T04:08:21.540490216Z [inf]  npm warn deprecated glob@7.2.3: Old versions of glob are not supported, and contain widely publicized security vulnerabilities, which have been fixed in the current version. Please update. Support for old versions may be purchased (at exorbitant rates) by contacting i@izs.me

2026-04-24T04:08:21.634720188Z [inf]  copy /usr/local/bin/mise
2026-04-24T04:08:21.635896059Z [inf]  copy /etc/mise/config.toml
2026-04-24T04:08:21.691841748Z [inf]  copy /etc/mise/config.toml
2026-04-24T04:08:21.692636442Z [inf]  copy /root/.local/state/mise
2026-04-24T04:08:21.710218951Z [inf]  copy /root/.local/state/mise
2026-04-24T04:08:22.03928133Z [inf]  npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead

2026-04-24T04:08:22.039535293Z [inf]  npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead

2026-04-24T04:08:23.48500978Z [inf]  npm warn deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.

2026-04-24T04:08:24.465417652Z [inf]  
added 472 packages, and audited 473 packages in 5s


2026-04-24T04:08:24.465432703Z [inf]  158 packages are looking for funding
  run `npm fund` for details

2026-04-24T04:08:24.481120321Z [inf]  
4 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

2026-04-24T04:08:24.482046515Z [inf]  npm notice
npm notice New major version of npm available! 10.8.2 -> 11.13.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.13.0
npm notice To update run: npm install -g npm@11.13.0
npm notice

2026-04-24T04:08:24.66576467Z [inf]  
> swiftshift@0.0.0 build
> tsc && vite build


2026-04-24T04:08:28.419862703Z [inf]  src/App.tsx(1369,10): error TS6133: 'workSchedule' is declared but its value is never read.

2026-04-24T04:08:28.420140062Z [inf]  src/App.tsx(1371,10): error TS6133: 'workAvailability' is declared but its value is never read.

2026-04-24T04:08:28.420194262Z [inf]  src/App.tsx(1837,9): error TS6133: 'periodHours' is declared but its value is never read.

2026-04-24T04:08:28.719231666Z [err]  sh build.sh
2026-04-24T04:08:28.734824655Z [err]  ERROR: failed to build: failed to solve: process "sh -c sh build.sh" did not complete successfully: exit code: 2



here's the render log: 
2026-04-24T04:08:30.874673829Z ==> Downloading cache...
2026-04-24T04:08:30.966907631Z ==> Cloning from https://github.com/GiTD67/SwiftShift
2026-04-24T04:08:31.48293505Z ==> Checking out commit 17ae63a7909abf054e9074e3c7029b452ec27d2c in branch main
2026-04-24T04:08:36.896789055Z ==> Downloaded 216MB in 4s. Extraction took 2s.
2026-04-24T04:08:42.268650808Z ==> Using Python version 3.11.0 via environment variable PYTHON_VERSION
2026-04-24T04:08:42.268664739Z ==> Docs on specifying a Python version: https://render.com/docs/python-version
2026-04-24T04:08:42.26883933Z ==> Installing Python version 3.11.0...
2026-04-24T04:08:56.21223234Z ==> Using Poetry version 2.1.3 (default)
2026-04-24T04:08:56.255801838Z ==> Docs on specifying a Poetry version: https://render.com/docs/poetry-version
2026-04-24T04:08:56.324595892Z ==> Running build command 'chmod +x build.sh && ./build.sh'...
2026-04-24T04:08:56.332481436Z Building frontend...
2026-04-24T04:08:56.44833232Z ==> Using Node.js version 24.14.1 (default)
2026-04-24T04:08:56.473360664Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version
2026-04-24T04:09:01.648499387Z 
2026-04-24T04:09:01.648527549Z added 472 packages, and audited 473 packages in 5s
2026-04-24T04:09:01.64854678Z 
2026-04-24T04:09:01.648660268Z 158 packages are looking for funding
2026-04-24T04:09:01.648787346Z   run `npm fund` for details
2026-04-24T04:09:01.660911542Z 
2026-04-24T04:09:01.660925543Z 4 moderate severity vulnerabilities
2026-04-24T04:09:01.660929963Z 
2026-04-24T04:09:01.660934424Z To address all issues (including breaking changes), run:
2026-04-24T04:09:01.660938974Z   npm audit fix --force
2026-04-24T04:09:01.660942674Z 
2026-04-24T04:09:01.660948935Z Run `npm audit` for details.
2026-04-24T04:09:01.879649138Z 
2026-04-24T04:09:01.879673309Z > swiftshift@0.0.0 build
2026-04-24T04:09:01.879678299Z > tsc && vite build
2026-04-24T04:09:01.87968247Z 
2026-04-24T04:09:05.627498798Z src/App.tsx(1369,10): error TS6133: 'workSchedule' is declared but its value is never read.
2026-04-24T04:09:05.627647728Z src/App.tsx(1371,10): error TS6133: 'workAvailability' is declared but its value is never read.
2026-04-24T04:09:05.627659098Z src/App.tsx(1837,9): error TS6133: 'periodHours' is declared but its value is never read.
2026-04-24T04:09:05.711036653Z ==> Build failed 😞
2026-04-24T04:09:05.711057744Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys

--- a/frontend/src/App.tsx
+++ b/frontend/src/App.tsx
@@ -1366,9 +1366,9 @@ export default function App() {
 
   // Profile tabs state
   const [profileTab, setProfileTab] = useState<'info' | 'schedule' | 'deposit' | 'availability'>('info')
-  const [workSchedule, setWorkSchedule] = useState<any>(null)
+  const [, setWorkSchedule] = useState<any>(null)
   const [directDeposit, setDirectDeposit] = useState<any>(null)
-  const [workAvailability, setWorkAvailability] = useState<any>(null)
+  const [, setWorkAvailability] = useState<any>(null)
   const [scheduleEdit, setScheduleEdit] = useState<any>(null)
   const [depositEdit, setDepositEdit] = useState<any>(null)
   const [availabilityEdit, setAvailabilityEdit] = useState<any>(null)
@@ -1834,7 +1834,6 @@ export default function App() {
   // Pay period
   const period = useMemo(() => payPeriodFor(now), [now])
   const periodLabel = `${period.start.toLocaleDateString([], { month: '2-digit', day: '2-digit', year: 'numeric' })} – ${period.end.toLocaleDateString([], { month: '2-digit', day: '2-digit', year: 'numeric' })}`
-  const periodHours = ((periodTotalMs + sessionWorkedMs) / 3600000).toFixed(1)
 
   // Overtime 1.5× earnings calculation for pay period sidebar
   const periodEarnings = useMemo(() => {

